app/models/turker/TurkerTable.scala

```diff
@@ -26,6 +31,18 @@ object TurkerTable{
   val db = play.api.db.slick.DB
   val turkers = TableQuery[TurkerTable]
 
+  def getAllTurkers : List[Turker] = db.withTransaction{ implicit session =>
+    turkers.list
+  }
+
+  def getConditionIdByTurkerId(turkerId: String): Option[Int] = db.withTransaction { implicit session =>
+    val cId = turkers.filter(_.turkerId === turkerId).list.headOption
```

Comment:
Try to use Scala's strong typing in all the code you write. Also rename this variable....
val turker: Turker = turkers.filter...